source/scpd/Cpp.d

```diff
@@ -235,6 +235,23 @@ extern(C++, (StdNS!())) struct vector (T, Alloc = allocator!T)
             return this._start[start .. end];
         }
 
+        public bool opEquals (in vector rhs) const pure nothrow @nogc @trusted
```

Comment:
Why the `@trusted` ?

Reply:
Well, `opIndex` is not `@trusted`. The question is, should it be? I'm not sure.

Reply:
> The question is, should it be? I'm not sure

It seems to offer a `@safe` interface when assertions are on.